corehq/apps/hqadmin/views.py

```diff
@@ -777,7 +777,7 @@ def view_data(case_id, indicators):
             'case': CommCareCase.get(case_id),
         }
 
-    if user or user_case:
+    if (user or user_case) and domain.call_center_config.use_fixtures:
```

Comment:
I don't think this is needed here since the entire purpose of this view is to view the fixtures.